### DISCUSSION TITLE
Timer Peripheral at MPFS-ICICLE-KIT-ES

### DISCUSSION THREAD

**Discussion by Linciclon** [polarfire-soc/polarfire-soc-documentation#258] [General] [answered]
Hello, I'm trying to use timer peripheral of MPFS-ICICLE-KIT-ES, described in "PolarFire ® SoC MSS Technical Reference Manual":
![Screenshot from 2022-11-29 17-50-02](https://user-images.githubusercontent.com/34252486/204605385-1726afb6-1f2c-499c-9686-3d6abb024f06.png)

However, I can't find the pdf: "PolarFire SoC Device Register Map" to get more information about this peripheral.
![Screenshot from 2022-11-29 17-23-34](https://user-images.githubusercontent.com/34252486/204605787-5cbadd29-d0fc-4057-b4b9-b81972212076.png)

Can you please help me to know how can I found this pdf ("PolarFire SoC Device Register Map") or to know what peripheral is this? In this way I can search for datasheet and configure this timer peripheral.
Thanks.

**Comment by gsriker77** [comment, marked as answer]
Looks like the above link is broken. I will check on it. Meanwhile you can download Polarfire SoC Register map from below link
https://ww1.microchip.com/downloads/aemDocuments/documents/FPGA/ProductDocuments/SupportingCollateral/v1_3_register_map.zip
(OR) 4.1.6.3 Register Descriptions of TRM -> Click 'Register map link'
https://ww1.microchip.com/downloads/aemDocuments/documents/FPGA/ProductDocuments/ReferenceManuals/PolarFire_SoC_FPGA_MSS_Technical_Reference_Manual_VC.pdf

Thanks,

**Comment by gsriker77** [comment]
We have latest Polarfire SoC Register map (v 1.5) at below link. Please refer below.
https://ww1.microchip.com/downloads/aemDocuments/documents/FPGA/ProductDocuments/SupportingCollateral/PolarFireSoC_Register_Map.zip

Thanks,
